src/views/Home/Home.jsx

```diff
@@ -1,8 +1,10 @@
 import { useEffect, useState } from 'react'
 import Profile from '../../components/Profile/Profile'
+import { useUser } from '../../context/UserContext'
```

Comment:
Using this here is redundant because you're still passing user as a prop from App.js